### PR TITLE
sample FeedInventory controller

### PR DESCRIPTION
**Pull request by yinsenyan**
Signed-off-by: yinsenyan <yinsenyan@tencent.com>

### CODE REVIEW

**Comment by yinsenyan** [comment]
@dixudx please review

File: cmd/sample-controller/main.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+package main
+
+import (
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	controller "github.com/clusternet/feedinventory-controller/pkg/controller"
```
Comment (dixudx) [inline cmd/sample-controller/main.go]: no such a module

File: cmd/sample-controller/main.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+	corev1 "k8s.io/api/core/v1"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/controller-manager/pkg/clientbuilder"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/scheme"
```
Comment (dixudx) [inline cmd/sample-controller/main.go]: I don't think we need to import `k8s.io/kubectl`.

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,138 @@
+module github.com/clusternet/feedinventory-controller
```
Comment (dixudx) [inline go.mod]: github.com/clusternet/sample-controllers

File: cmd/sample-controller/main.go (hunk 1 above)
Comment (yinsenyan) [reply cmd/sample-controller/main.go]: will replace to sample-controller

File: cmd/sample-controller/main.go (hunk 2 above)
Comment (yinsenyan) [reply cmd/sample-controller/main.go]: error import , will replace to "k8s.io/client-go/kubernetes/scheme"

**Review by dixudx — APPROVED** [review]

**Pull request merged**